src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1333,9 +1333,37 @@ def post_process(self, r):
                 while r is not None and r["properties"]["provisioningState"].lower() == "inprogress":
                     r = self.client.show(self.cmd, self.get_argument_resource_group_name(), self.get_argument_name())
                     time.sleep(1)
-                linker_client.linker.begin_create_or_update(resource_uri=r["id"],
-                                                            parameters=item["parameters"],
-                                                            linker_name=item["linker_name"]).result()
+
+                service_connectors_def_list, service_bindings_def_list = parse_service_bindings(
```

Comment:
May I ask why it need to `parse_service_bindings` again in `post_process`? As it should has been parse in `set_up_service_binds` in `construct_payload` .

Reply:
In `set_up_service_binds`, we already set the parse result of `service_connectors_def_list` in context, which will be used in `post_process`.  If you need to add `service_bindings_def_list` to context, here is the sample:
```
def set_argument_service_connectors_def_list(self, service_connectors_def_list):
        self.set_param("service_connectors_def_list", service_connectors_def_list)
def set_argument_service_bindings_def_list(self, service_bindings_def_list):
        self.set_param("service_bindings_def_list", service_bindings_def_list)
```

Reply:
1. `set_up_service_binds` and `set_up_service_bindings` are for dev services (these are in the form of a container app). in `post_process` both in create and update, we are setting up managed bindings (which are for managed services and these are non-containerapp services -- postgresql, mysql, kafka). 
2. the setups are pretty different with managed bindings using a linker object while the dev bindings are checked from the container app spec. they need to all be unique so that's why the `check_unique_bindings` logic is duplicated.
3. i'll revisit the def_list part ...i kinda see what you mean but i want to make sure i totally understand.